tailwind.config.js

```diff
@@ -0,0 +1,39 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.html"],
+  theme: {
+    extend: {
+      spacing: {
+        maxScreen: "calc((100% - 1480px) / 2)",
```

Comment:
This name is not descriptive, read the following for calculating margin and width, it also include some css rules 

> ## CSS Rules

### Units of measurement

- Use `rem` whenever possible, and avoid using `px` as a measurement unit
- Translate the Figma designs to `rem`, by dividing the pixels by 16 ( `1rem` equal to `16px`)
- Use line-height ratio (which has no measurement) instead of `px`. To calculate it use the line-height in `px` and divide it by the font-size in `px`.
- page margin: convert the left-right padding of the pages to the content width. this is the formula to calculate it:

```
content_width = (figma_page_width - (padding_of_one_of_the_sides * 2)) * 100 / figma_page_width

content_width = (1264) * 100 / 1920
```

- When dealing with mobile devices (iOS and Android), in case you would want an element to occupy just the total height minus the address bar. You may consider using `100svh` instead of `100vh` as the CSS height value.
    - In tailwind CSS the `h-screen` class should give you a height of `100vh`, however, the `100svh` class is not present. you have to add it manually.